FE-061C-Create-Report-Page

hmsye opened the pull request:
**Created 'Report' page within the 'Community' section.

hmsye committed:
Report button added

hmsye committed:
Button Label changes

hmsye committed:
Created 'Report page' within the Community section

hmsye committed:
Created 'Report page' within the Community section

hmsye committed:
Created 'Report page' within the Community section

ArjunBhardwaj3 commented:
This was code reviewed by Arjun: Overall it is ready to be merged but there are some issues that need to be corrected, 
the back button redirects into settings but it should be to home page or you can completely remove it (make sure if a user clicks on community from under settings then it should redirects to settings but if we open from navigation bar it should go home).  
The report doesn't work from the education page. Also, after the risk meter page then the navigation bar has no texts. Last point, after a user clicks on report it highlights home on the navigation bar instead of report.

hmsye committed:
Fixed issues relating to the functionality of the 'back' button in the 'Report' page.
- If user access 'Report' through Settings > Community, pressing the 'back button will return them to Settings'.
- If user accesses 'Report' via the bottom nav, pressing the 'back' button will return the to 'Home'.
- Going between the tabs in the 'Community' section and pressing 'back' on the 'Report' page will return the user to Settings.
- Report button now works from the Education section.

hmsye committed:
-bug fix

hmsye committed:
Rebase - dev

hmsye committed:
Revert "-bug fix"

This reverts commit 2fc717abf3bc1edb69ac221978bc98c37711ed4d.

hmsye committed:
Revert "Rebase - dev"

This reverts commit f5e88e6bc5e2fd55ba75b06bbea047fcc467303e.

hmsye committed:
fix

hmsye committed:
-bug fixes

hmsye committed:
-'Report' button now works from Risk Scanner and its related activities.
- Bottom Navigation Bar labels will show on all screens.
- Bottom Navigation Bar labels and icons are aligned on all screens.
- 'Report' will be highlighted when user accessing the 'Report' page.

aroomakhann committed:
placed the back button pop-up back in to not overwrite work

aroomakhann commented:
Thank-you Hassan!
